invoicing/sap/fetch/process.py

```diff
@@ -32,7 +32,7 @@ def get_payment_date(self) -> datetime.date:
         return datetime.strptime(self.get_value_from_line(15, 6), "%d%m%y").date()
 
     def get_amount(self) -> Decimal:
-        return Decimal(int(self.get_value_from_line(79, 10)) / Decimal("100")).quantize(
+        return Decimal(int(self.get_value_from_line(77, 10)) / Decimal("100")).quantize(
```

Comment:
I don't understand how this was changed, since it used to work and was tested with the files that we got from SAP test server.

Did they make changes on SAP end?  Or has it just always been broken and nobody has noticed that before?

Wouldn't this read the amount completely incorrectly? If I understand correctly, the code would have read 123456789055 as 34567890.55 instead of 12345678.90 (where 55 would be in next field).

Reply:
All payments have an extra 0, they are all incorrect.

Reply:
You must be right. This just hasn't been tested properly until now. There was big problems getting connections to work (because of firewalls etc) when this was supposed to be tested and I guess that the customer just didn't get the tests completed back then.

Documentation would be good. What is the extra digit at the end? All files which I have seen have either 0 or 1 at that place so it quite probable is not part of the amount.

Anyway, it seems that this change should be correct. And if customer has reported that the all of the current amounts are incorrect, then at least it won't make things worse for sure.

Reply:
Just fix the tests please and this would be good to go IMHO. Thanks!

Reply:
Done..